src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/ExpressionCompilerTestBase.cs

```diff
@@ -293,8 +293,7 @@ internal static Alias ObjectIdAlias(uint id, string typeAssemblyQualifiedName)
         {
             Assert.NotEqual(0u, id); // Not a valid id.
             var name = $"${id}";
-            var fullName = id.ToString();
-            return new Alias(DkmClrAliasKind.ObjectId, name, fullName, typeAssemblyQualifiedName, default(CustomTypeInfo));
+            return new Alias(DkmClrAliasKind.ObjectId, name, name, typeAssemblyQualifiedName, default(CustomTypeInfo));
```

Comment:
Since we're assuming that ```name == fullName``` for the purposes of testing, it might be good to add a ```Debug.Assert(name == fullName)``` to ```ExpressionCompiler.GetAliases``` in the ObjectId case.